example_webserver/example.py

```diff
@@ -47,6 +56,11 @@ async def processing_busy(request):
     return PlainTextResponse(result)
 
 
+def processing_with_yield(_request):
+    result = _process_request_with_yield()
```

Comment:
This needs to be an async function, and you need to await _process_request_with_yield()